src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -122,8 +123,8 @@ public async Task UpdateSearchParameterStatusAsync(IReadOnlyCollection<string> s
 
         internal async Task AddSearchParameterStatusAsync(IReadOnlyCollection<string> searchParamUris)
         {
-            // new search parameters are added as supported, until reindexing occurs, when
-            // they will be fully enabled
+            // new search parameters are added as supported, however the UpdateSearchParameterStatus will
+            // check the search parameter to ensure the fhirpath is valid and supported
```

Comment:
nit: should we just revert this file? I don't think this new comment is right anymore.

Reply:
Yep!  I think it is back to way it was.